content/en/Integrations/IntegrationBuilder/configure-gitlab-integration.md

```diff
@@ -0,0 +1,138 @@
+
+Follow [GitLab Docs](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html) to learn more about creating a project access token.
+
+### 3. Create a GitLab Connection on the Cobalt Platform
+
+From now on, we are working on the Cobalt platform side.
+
+Go to your integration builder page, and select your project.
+
+{{% image src="/integrations/integration_builder/gitlab/01-select-project.png" alt="Select project" %}}
+
+Click on the "Create" button on the top right, and choose "Connection".
+
+{{% image src="/integrations/integration_builder/gitlab/02-create-connection.png" alt="Create a new connection" %}}
+
+You will be redirected to a page with all the possible connections.
+
+{{% image src="/integrations/integration_builder/gitlab/03-all-connections.png" alt="All possible connections" %}}
+
+Find the GitLab connection and click on it.
+
+{{% image src="/integrations/integration_builder/gitlab/04-find-gitlab-connection.png" alt="Find GitLab connection" %}}
+
+You will need to fill in some information about the connection. The most important part is the access token you obtained in the first step.
+
+{{% image src="/integrations/integration_builder/gitlab/05-gitlab-connection-data.png" alt="Create GitLab connection" %}}
+
+Alternatively, you can use password-based authentication. In this case, you need to provide your GitLab username and password.
+
+{{% image src="/integrations/integration_builder/gitlab/06-gitlab-connection-auth-type.png" alt="GitLab auth types" %}}
+
+Once done, click "Connect".
+
+{{% image src="/integrations/integration_builder/gitlab/08-connection-connected.png" alt="GitLab connection successful" %}}
+
+### 4. Create a New Folder to Better Organize Your Recipes
+
+Before creating a new recipe, it is better to create a new folder for all future GitLab recipes.
+
+Go to the project page again, and click on the "Create" button on the top right. This time, choose "Folder".
+
+{{% image src="/integrations/integration_builder/gitlab/09-new-folder.png" alt="Create a new folder" %}}
+
+Give the new folder a name, and choose its location. Then click "Create Folder".
+
+{{% image src="/integrations/integration_builder/gitlab/10-new-folder-data.png" alt="New folder data" %}}
+
+Now you can see the new folder in the project.
+
+{{% image src="/integrations/integration_builder/gitlab/11-folder-created.png" alt="New folder created" %}}
+
+Now we have the option in the folder to create a new recipe. But it is better to reuse an existing recipe.
+
+### 5. Find an Existing Recipe
+
+Go to the "Library" tab and search for the GitLab recipe.
+
+{{% image src="/integrations/integration_builder/gitlab/12-find-community-recipe.png" alt="Find GitLab recipe" %}}
+
+There is no GitLab recipe found for a new pentest finding, but a recipe for a DAST finding exists. We can use it as a base and customize it. Click on the recipe to see its details and click "Use this recipe".
```

Comment:
My understanding is that a "pending fix" pentest finding and "detected" DAST finding are equivalent. They are both visible to the user when they look into the platform. For contrast, I'm pretty sure a "new" pentest finding is only visible to pentesters, so it wouldn't make much sense to have an issue created for these.
```suggestion
There are 2 recipes to choose from, depending on whether you're interested in creating GitLab issues to track pentest or DAST findings. We'll use the DAST recipe as a base and customize it. Click on the recipe to see its details and click "Use this recipe".
```